src/scene/scene.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __SCENE_H__
+#define __SCENE_H__
+
+#include <vector>
+#include <glm/glm.hpp>
+
+#include "triangle.h"
+
+class Scene
+{
+  unsigned int mTriangleCount;
```

Comment:
so I know classes are default private. Even so I like to explicitly mark member properties that should be private as private.